### PR TITLE
Add documentation about UTC time zone being fixed for SQL DB and Synapse Analytics

### PR DESCRIPTION
**Pull request by goventur**

### CODE REVIEW

**Comment by PRMerger17** [comment]
@goventur : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@markingmyname @julieMSFT - Could you review the proposed updates? If approved, could you verify whether the new paragraph is intended to be part of the note or plain text? Thanks.

**Comment by markingmyname** [comment]
@goventur 

We are verifying this change.

Thank you.

**Comment by julieMSFT** [comment]
@XiaoyuMSFT @mstehrani Hi Emily and Xiaoyu, Please review the changes suggested in this PR and approve if the changes are good. Thank byou.

**Comment by markingmyname** [comment]
@goventur,

Please review the suggested changes.

thank you.

File: docs/t-sql/functions/getdate-transact-sql.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ Returns the current database system timestamp as a **datetime** value without th
 > [!NOTE]
 >  SYSDATETIME and SYSUTCDATETIME have more fractional seconds precision than GETDATE and GETUTCDATE. SYSDATETIMEOFFSET includes the system time zone offset. SYSDATETIME, SYSUTCDATETIME, and SYSDATETIMEOFFSET can be assigned to a variable of any of the date and time types.
 
+Azure SQL Database and Azure Synapse Analytics do not support time zone settings; it always follows UTC. Use [AT TIME ZONE](../../t-sql/queries/at-time-zone-transact-sql.md) in Azure SQL Database or Azure Synapse Analytics if you need to interpret date and time information in a non-UTC time zone.
```
[the remark this answers is not in the document]
Comment (goventur) [reply docs/t-sql/functions/getdate-transact-sql.md:42]: Totally agree with the changes.
Thanks!

File: docs/t-sql/functions/sysdatetime-transact-sql.md [hunk 2]
```diff
@@ -38,6 +38,8 @@ monikerRange: ">=aps-pdw-2016||=azuresqldb-current||=azure-sqldw-latest||>=sql-s
 > [!NOTE]  
 >  SYSDATETIME and SYSUTCDATETIME have more fractional seconds precision than GETDATE and GETUTCDATE. SYSDATETIMEOFFSET includes the system time zone offset. SYSDATETIME, SYSUTCDATETIME, and SYSDATETIMEOFFSET can be assigned to a variable of any of the date and time types.  
   
+Azure SQL Database and Azure Synapse Analytics do not support time zone settings; it always follows UTC. Use [AT TIME ZONE](../../t-sql/queries/at-time-zone-transact-sql.md) in Azure SQL Database or Azure Synapse Analytics if you need to interpret date and time information in a non-UTC time zone.
```
[the remark this answers is not in the document]
Comment (goventur) [reply docs/t-sql/functions/sysdatetime-transact-sql.md:41]: Totally agree with the changes.
Thanks!

**Comment by goventur** [comment]
Reviewed and agree with the changes.
Thanks!

**Comment by ktoliver** [comment]
@markingmyname - Would you like the PR review team to make the changes in the PR and then merge?

**Comment by julieMSFT** [comment]
@Ktoliver Yes, please make the changes to the PR and merge. Thank you!

**Comment by PRMerger18** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@julieMSFT - I changed _with exception to Azure SQL Managed Instance_ to _with the exception of Azure SQL Managed Instance_ -- OK?

**Comment by mstehrani** [comment]
@ktoliver Yes that's fine, thank you!

**Comment by julieMSFT** [comment]
Yes, thank  you Kristine!

Thank you,

Julie

**Pull request merged**